doc/source/index.rst

```diff
@@ -45,11 +45,11 @@ Quickstart
         containers:
         - image: daskdev/dask:latest
           imagePullPolicy: IfNotPresent
-          args: [dask-worker, --nthreads, '2', --no-bokeh, --memory-limit, 6GB, --death-timeout, '60']
+          args: [dask-worker, $(DASK_SCHEDULER_ADDRESS), --nthreads, '2', --no-dashboard, --memory-limit, 6GB, --death-timeout, '60']
```

Comment:
I don't think the `$(DASK_SCHEDULER_ADDRESS)` syntax is correct here, the brackets mean the value of the variable will be executed. 

Also the worker should be able to pick up this environment variable itself and shouldn't need specifying in the CLI if set. 

Can we drop this again or are you have specific issues with this?